packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -1277,21 +1277,15 @@ class _CurvedAnimationBuilderState extends State<_CurvedAnimationBuilder> {
   // direction.
   void _updateStatus(AnimationStatus status) {
     if (_animationDirection != status) {
-      setState(() {
-        _animationDirection = status;
-      });
+      setState(() => _animationDirection = status);
```

Comment:
In general, we prefer styling `setState` as a block like it was before. Can you revert this formatting change?

Reply:
same below.

Reply:
I had a look through the repo and you're right—an overwhelming majority of `setState` calls use a function body even if there's just one statement.

I was surprised at how many were formatted as a single line without the `=>` syntax:

```dart
setState(() { _value = newValue; });
```

I guess now I know to use curly braces in the Flutter repo from now on.